src/platform/datapath_raw_xdp.c

```diff
@@ -428,40 +428,108 @@ CxPlatDpRawGetDapathSize(
     return sizeof(XDP_DATAPATH);
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+CxPlatDpRawInterfaceUninitialize(
+    _Inout_ XDP_INTERFACE* Interface
+    )
+{
+
+    #pragma warning(push)
+    #pragma warning(disable:6001) // Using uninitialized memory
```

Comment:
What's this needed for?

Reply:
the compiler is confused and it thinks `Interface->Queues` wasn't initialized but we zero it before passing it to this function. I copied these pragma from the existing code. They were in CxPlatDpRawUninitialize.